pkgs/applications/science/logic/z3/default.nix

```diff
@@ -27,9 +27,9 @@ stdenv.mkDerivation rec {
     mv $out/lib          $lib/lib
     mv $out/include      $dev/include
 
-    # clean up a copy of libz3.so and symlink it instead
-    rm $python/${python.sitePackages}/z3/lib/libz3.so
-    ln -s $lib/lib/libz3.so $python/${python.sitePackages}/z3/lib/libz3.so
```

Comment:
We have `hostPlatform.extensions.sharedLibrary` to handle conditional like this.

Reply:
👍 updated